exploration/open-close-placeholders.md

```diff
@@ -1,4 +1,4 @@
-# Open/Close Expressions
+# Open/Close Placeholders
```

Comment:
I think we would be better off calling this design "spannable" but don't care *that* much about it.

Reply:
All of the options currently being considered are `placeholders` in a pattern; hence my suggested name for this.

Reply:
+1 to titling this design doc "Spannables". If you agree, please also rename the file.

Reply:
I would prefer "placeholders".